airbyte-integrations/connectors/source-stripe/main.py

```diff
@@ -8,9 +8,10 @@
 from airbyte_cdk.entrypoint import AirbyteEntrypoint, launch
 from source_stripe import SourceStripe
 
+
 if __name__ == "__main__":
     args = sys.argv[1:]
-    state = AirbyteEntrypoint.extract_state(args)
-    use_concurrent_cdk = state is None
-    source = SourceStripe(use_concurrent_cdk=use_concurrent_cdk)
+    state = SourceStripe.read_state(AirbyteEntrypoint.extract_state(args))
+    catalog = SourceStripe.read_catalog(AirbyteEntrypoint.extract_catalog(args))
```

Comment:
We might want to update `singer/source.py` because `read_catalog` is not classmethod (or is singer still used and could we just delete it)?

Reply:
preference for updating it (assuming its CATs pass)